src/app/static/src/app/components/modelCalibrate/CalibrationResults.vue

```diff
@@ -1,4 +1,7 @@
 <template>
+    <hr />
+    <h3 v-translate="'calibrateResultsHeader'"></h3>
+    <p class="text-muted" v-translate="'calibrateResultsDesc'"></p>
```

Comment:
Bit confused as to why this is needed, does it fix any tests?

Reply:
Nope, just brings this up to same as current prod